#Print Function

(Writing it with # before the name of section makes it as a section that helps in enclosing, declosing the section cells by top-down arrows)

In [7]:
print("Ajay rao") #printing

Ajay rao


In [9]:
print("Hello" + " Everyone!") #printing by concatenating the strings

Hello Everyone!


In [5]:
print(10) #printing a number
type(10)  #gives datatype

10


int

In [6]:
print(2+9) #printing the addition of two numbers (used addition operator '+')

11


Basic data types : int,float,str

In [11]:
print(type(10)) #int
print(type(12.5)) #float
print(type('Ajay')) #str

<class 'int'>
<class 'float'>
<class 'str'>


#Constants & Variables

In [12]:
super hero="Iron Man"
print(super hero)

SyntaxError: invalid syntax (<ipython-input-12-ca664f2c5808>, line 1)

To avoid such error should make variable declared (2 worded) as a single word variable by using symbols such as '_' in between words or by avoiding space between words

In [18]:
super_hero="Iron Man"
print(super_hero)
#or
superhero="Iron Man"
print(superhero)

Iron Man
Iron Man


In [15]:
a,b,c="CSE","ECE","AIML" #can be declared together
#can be printed individually
print(a)
print(b)
print(c)
print(a,b,c) #can be printed altogether

CSE
ECE
AIML
CSE ECE AIML


In [17]:
x=y=z=10 #same value can be assigned together
print(x)
print(y)
print(z)
print(x,y,z)

10
10
10
10 10 10


#Input Function


In [19]:
n1=input("Enter a number : ")
n2=input("Enter a number : ")
sum=n1+n2
print(sum)

Enter a number : 10
Enter a number : 5
105


Here above input is considereed as a string by default so 10+5 return 105 as it concatenates a string

To get sum we can write as below :

In [20]:
n1=int(input("Enter a number : ")) #giving int beforehand makes it to consider as a number
n2=int(input("Enter a number : "))
sum=n1+n2
print(sum)

Enter a number : 10
Enter a number : 5
15


In [26]:
#changing the datatype in python:
n=5
print(n)
print(type(n))
n=float(n) #changing to float
print(n)
print(type(n))

5
<class 'int'>
5.0
<class 'float'>
